### PR TITLE
Contains all updates for PLAT-94872

### PR DESCRIPTION
**Pull request by jbranson000**
Contains all updates for PLAT-94872

### CODE REVIEW

File: components/mixins/profile/profile-mobile-plan-details.schema.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile/profile-mobile-plan-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Mobile Plan Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:tags": { "industry": ["telecom"] },
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
```
Comment (mjasrotia) [inline components/mixins/profile/profile-mobile-plan-details.schema.json:15]: Is this supposed to extend profile ?

File: components/mixins/profile/profile-tax-profile-details.schema.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile/profile-tax-profile-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Personal Finance Details",
```
Comment (mjasrotia) [inline components/mixins/profile/profile-tax-profile-details.schema.json:10]: Should this be more elaborative like "Personal Tax Profile" or something like that ?

File: components/mixins/experience-event/industry-verticals/experienceevent-device-details.schema.json [hunk 3]
```diff
@@ -32,20 +32,20 @@
...
               "$ref": "https://ns.adobe.com/xdm/datatypes/tool-usage",
               "description": "Extension of toolUsage data type.",
-			  "meta:status": "deprecated"
+              "meta:status": "deprecated"
             },
-            "xdm:deviceTradeInManufacturer": {
+            "xdm:manufacturer": {
               "title": "Manufacturer",
               "type": "string",
               "description": "Manufacturer of the device being traded."
```
Comment (hanessian) [inline components/mixins/experience-event/industry-verticals/experienceevent-device-details.schema.json:40]: is this in reference to a traded device anymore? @jbranson000. If not let's remove "being traded."

File: components/mixins/experience-event/industry-verticals/experienceevent-device-details.schema.json [hunk 4]
```diff
@@ -23,7 +23,7 @@
           "title": "Device Configuration",
           "$ref": "https://ns.adobe.com/xdm/datatypes/tool-usage",
           "description": "Extension of toolUsage data type.",
-		  "meta:status": "deprecated"
+          "meta:status": "deprecated"
         },
         "xdm:deviceTradeInDetails": {
```
Comment (mjasrotia) [inline components/mixins/experience-event/industry-verticals/experienceevent-device-details.schema.json:28]: should we call this object "mobileDevice" ?

File: components/mixins/profile/profile-mobile-plan-details.schema.json (hunk 1 above)
Comment (jbranson000) [reply components/mixins/profile/profile-mobile-plan-details.schema.json:15]: Yes

File: components/mixins/experience-event/industry-verticals/experienceevent-device-details.schema.json (hunk 3 above)
Comment (jbranson000) [reply components/mixins/experience-event/industry-verticals/experienceevent-device-details.schema.json:40]: This is still valid and part of the deviceTradeInDetails node, so let's leave as-is for now

File: components/mixins/profile/profile-tax-profile-details.schema.json (hunk 2 above)
Comment (jbranson000) [reply components/mixins/profile/profile-tax-profile-details.schema.json:10]: I'll rename it with the next commit.

File: components/mixins/profile/profile-mobile-plan-details.schema.json (hunk 1 above)
Comment (hanessian) [reply components/mixins/profile/profile-mobile-plan-details.schema.json:15]: +1 to changing to profile. Thanks!

File: components/mixins/profile/profile-personal-tax-profile-details.schema.json [hunk 5]
```diff
@@ -0,0 +1,66 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/profile/profile-personal-tax-profile-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Personal Tax Profile Details",
```
Comment (hanessian) [inline components/mixins/profile/profile-personal-tax-profile-details.schema.json:10]: Might be a little redundant with "personal tax profile details." Can we switch to tax profile details? Personal is implicit in it being part of profile class. Thanks! @jbranson000

File: components/mixins/profile/profile-user-account-details.schema.json [hunk 6]
```diff
@@ -61,13 +61,13 @@
               "type": "string",
               "description": "Captures the user account type, if applicable."
             },
-            "xdm:preferences": {
-              "title": "Preferences",
+            "xdm:shoppingPreferences": {
+              "title": "Shopping Preferences",
```
Comment (hanessian) [inline components/mixins/profile/profile-user-account-details.schema.json:65]: Is there a way we can make this more generic without having shopping in front? I want tit to apply for travel/hosp also. So bed prefs and seating prefs. Would product preferences be more generic?

File: components/mixins/profile/profile-personal-tax-profile-details.schema.json [hunk 7]
```diff
@@ -0,0 +1,66 @@
+{
+  "meta:license": [
```
Comment (hanessian) [inline components/mixins/profile/profile-personal-tax-profile-details.schema.json:2]: Can this be an object under personal finances? So, personalFinances.taxProfile.fields? That way it'll all be under the personal finances folder, but neatly in a tax object.

File: components/mixins/profile/profile-user-account-details.schema.json (hunk 6 above)
Comment (jbranson000) [reply components/mixins/profile/profile-user-account-details.schema.json:65]: categoryPreferences?

File: components/mixins/profile/profile-personal-finance-details.schema.json [hunk 8]
```diff
@@ -53,48 +82,15 @@
...
-              "description": "Refers to married couples who choose to record their respective incomes, exemptions, and deductions on separate tax returns."
-            },
-            "xdm:financialAccount": {
-              "title": "financeAccountId",
-              "type": "array",
-              "items": {
-                "$ref": "https://ns.adobe.com/xdm/datatypes/financial-account"
-              },
-              "description": "Financial account IDs assigned to the user, and array of Financial Account Details data type."
+            "xdm:ID": {
```
Comment (hanessian) [inline components/mixins/profile/profile-personal-finance-details.schema.json:85]: Should this still be an array? Couldn't they have multiple accounts we'd want to collect?

Comment (jbranson000) [reply components/mixins/profile/profile-personal-finance-details.schema.json:85]: just an array of Account IDs?  That should work.

File: components/mixins/profile/profile-personal-finance-details.schema.json [hunk 9]
```diff
@@ -53,48 +82,15 @@
...
+              "title": "ID",
+              "type": "string",
+              "description": "Financial account ID assigned to the user."
```
Comment (hanessian) [inline components/mixins/profile/profile-personal-finance-details.schema.json:88]: For taxid, should we include it incase it's relevant? For example if someone like an immigrant doesn't have a SSN?

**Review by mjasrotia — APPROVED** [review]

**Review by prabhum2 — APPROVED** [review]

**Pull request merged**